core/l10n_init.js

```diff
@@ -61,6 +61,8 @@ define('core/l10n_init',
     }
 
     function _transformLocale(locale) {
+        // Use default locale "en-US" if given locale is undefined (BUG 1138558)
+        locale = locale || 'en-US';
```

Comment:
No i totally i agree. i didn't want to offense you if you think that, i just tried to help. i also didn't find this bug, i just picked it from the bugzilla page as a start. i don't know the codebase of the project well, i just had a small look yet. i don't think that the changes in l10n.js are needed or great or smth like this. i even don't think that the changes in l10n_init.js are needed because i can't reproduce them on the production server, so they may be dev only too, but i have no idea why it only appears locally, even on the v2.5.0 tag.